### ISSUE TITLE
Damage done to shields is inconsistent between 91.0 and 98+

### ISSUE THREAD

**Issue by lhog** [ZeroK-RTS/Zero-K#663]
Recently I've noticed that shieldball lost its previous strength. It was especially evident when shielded units were attacked by moderators.

So I've just done an experiment on 98.x.x.x-451 and 91 to compare how a fully charged thug takes a single moderator shot.

Fully charged thug has following stat:  1250 shieldpower, 960 health, 0% slowness.

Net result after a shot has been fired:
98+: 263 shieldpower, 460 health(!), 50% slow (!)
91: 262 shieldpower, 960 health(!), 0% slow(!)

My sight didn't fail me, shields either take too much damage on 98+ or too little on 91.0

**Comment by Anarchid** [comment]
> 98+: 263 shieldpower

This looks like the damage was applied to shield *and* to the unit.

**Comment by GoogleFrog** [comment]
Damage done to shields is unchanged. What has changed is damage to units behind shields.

Fixed when
https://github.com/spring/spring/pull/180 is pulled
and 
https://springrts.com/mantis/view.php?id=4715 is fixed.

A workaround is to de-impactOnly all BeamLasers and LightningCannons.

**Comment by GoogleFrog** [comment]
I guess the workaround is fairly easy to add and remove (remove impactOnly from lasers which have it and set AoE really low). PRs seem to take a few days to go through and https://springrts.com/mantis/view.php?id=4715 seems to have stalled. So I will put a workaround in.

**Comment by Anarchid** [comment]
We wouldn't be able to use the PR anyway until the ati mapshaders issue is resolved.

**Comment by GoogleFrog** [comment]
What is the new ati mapshaders issue?

**Comment by Anarchid** [comment]
#646 / https://springrts.com/mantis/view.php?id=4715

**Comment by GoogleFrog** [comment]
Oh, I thought you were talking about a new issue because I linked that issue and talked about it in the two comment prior to yours.

The PR to fix this engine side was pulled.

**Comment by GoogleFrog** [comment]
Workaround is gameside and the removal is simply waiting on the adoption of latest engine.
